systemd/ceph-osd@.service.in

```diff
@@ -12,7 +12,14 @@ Environment=CLUSTER=ceph
 ExecStart=/usr/bin/ceph-osd -f --cluster ${CLUSTER} --id %i --setuser ceph --setgroup ceph
 ExecStartPre=/usr/lib/ceph/ceph-osd-prestart.sh --cluster ${CLUSTER} --id %i
 ExecReload=/bin/kill -HUP $MAINPID
+LockPersonality=true
+MemoryDenyWriteExecute=true
+NoNewPrivileges=true
```

Comment:
I think we need this one.  the osd and mon sudo smartctl (see ceph.git/sudoers.d)

Reply:
done